stages/org.osbuild.zip

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+import os
+import subprocess
+import sys
+
+import osbuild.api
+
+
+def main(inputs, output_dir, options):
+    tree = inputs["tree"]["path"]
+    filename = options["filename"].lstrip("/")
+    # The zip command itself uses 6 as default if no level is specified.
+    # We choose to specifiy 6 as default here to make it explicit.
+    level = options.get("level", 6)
+    paths = options.get("paths", [])
```

Comment:
Maybe name this "include" just like it's called in the zip man-page? It would also make it slightly clearer (IMHO) that this is takes globs as inputs, i.e. `-i subdir/` will not work, but `-i subdir/*` does.